tests/test-browser/chrome-test.js

```diff
@@ -0,0 +1,61 @@
+const assert = require('assert');
+const puppeteer = require('puppeteer');
+const inspect = require('util').inspect;
+
+const URL = process.argv[2];
+
+async function main() {
```

Comment:
Would benefit from some abstraction - This way the bootstrapping code is out of the way and we're just adding more tests to the `testGit` method.

```
function main() {
   bootstrapBrowser() 
   testGit()
}

function testGit() {
  validateGitClone();
  validateGitBranch()
 ....
}
```